vllm/v1/worker/gpu_input_batch.py

```diff
@@ -220,12 +220,11 @@ def __init__(
         self.repetition_penalties_reqs: set[str] = set()
 
         # Speculative decoding
-        self.num_accepted_tokens_cpu_tensor = torch.ones((max_num_reqs, ),
-                                                         dtype=torch.int64,
-                                                         device="cpu",
-                                                         pin_memory=pin_memory)
-        self.num_accepted_tokens_cpu = \
-            self.num_accepted_tokens_cpu_tensor.numpy()
+        self.num_accepted_tokens_cpu = torch.ones((max_num_reqs, ),
+                                                  dtype=torch.int64,
+                                                  device="cpu",
+                                                  pin_memory=pin_memory)
+        self.num_accepted_tokens_event = torch.cuda.Event()
```

Comment:
I think we'll want to keep the numpy view here because element-wise access is much faster than for cpu torch tensors.